.flake8

```diff
@@ -60,13 +60,15 @@ per-file-ignores =
   # WPS412 logic of an extension is in __init__.py file;
   # FIXME: WPS201 too many imports
   # FIXME: WPS402 too many `noqa`s
-  src/sphinxcontrib/towncrier/__init__.py: E800, WPS201, WPS402, WPS412
+  # FIXME: WPS202 too many module members
```

Comment:
If we just ignore every linting rule that annoys us, this would make using the linters pointless.
There's no good reason for ignoring this. If the rule is hit, this calls for splitting a module into two or more. Just like it was done with `_towncrier`.
Besides, it's configurable via `max-module-members` setting that defaults to `7` currently: https://wemake-python-styleguide.readthedocs.io/en/latest/pages/usage/violations/complexity.html#wemake_python_styleguide.violations.complexity.TooManyModuleMembersViolation.

Reply:
I am uncertain how you would rather them be split. I personally dont see the need for an extra module just for this one function which will only really be used under this specific context. `_towncrier` seem to contain a function that aids with getting the configuration, but other functions like `_lookup_towncrier_fragments` and `_get_changelog_draft_entries` are placed in the `__init__`. What would be the prefered criteria for when to split/how to place them?

Reply:
I'm not sure about the criteria but I'd be fine with putting a single function into a separate module.
There's another (hidden?) problem here — it is an antipattern to have the logic in an `__init__.py` module because this sort of thing causes side-effects during import-time. And so it's usually better to have an entry point in some `.api` submodule.
Obviously, it's not to be solved in this PR but it'd be good to do it at some point in the future.

FWIW I'll try to come up with an intermediate solution myself for now.